src/AppInstallerRepositoryCore/Rest/RestSourceFactory.cpp

```diff
@@ -4,6 +4,7 @@
 #include "RestSourceFactory.h"
 #include "RestClient.h"
 #include "RestSource.h"
+#include <Rest/HttpClientHelper.h>

```

Comment:
><Rest/HttpClientHelper.h> [](start = 9, length = 25)

Should be "HttpClientHelper.h", by the way, I don't see it being used in this cpp

Reply:
Yeah, i removed the usage and forgot to remove the header.

---
In reply to: [612859244](https://github.com/microsoft/winget-cli/pull/861#discussion_r612859244) [](ancestors = 612859244)